composer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "jeroen/mediawiki-github",
```

Comment:
No it is installing at `/extensions/MediaWikiGitHub` so probable the name should be changed to `"jeroen/github". I think it will be nicer to have this as `"mediawiki/gibhub"` as before.